mdbook-course/Cargo.toml

```diff
@@ -17,6 +17,6 @@ matter = "0.1.0-alpha4"
 mdbook = "0.4.40"
 pretty_env_logger = "0.5.0"
 regex = "1.10"
-serde = "1.0.208"
-serde_json = "1.0.125"
+serde = "1.0.209"
+serde_json = "1.0.127"
```

Comment:
When I see this, I wonder if we can configure the bot to not only update the lock file?

We don't actually depend on these patch version changes, so I think we don't need to touch all the `Cargo.toml` files all the time.

An easy alternative step would be to change the frequency to monthly, I don't know what you think of that?